old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/08-clean-data-bad-and-missing-values.Rmd

```diff
@@ -199,18 +217,27 @@ be considered a "bad" or miscalculated value.
 of 1110 is clearly not correct.
 
 ### Find Bad Data Values
-Sometimes a dataset's metadata will tell us the range of expected values for a
-variable or common sense dictates the expected value (as in the Julian day example above).
-Values outside of this range are suspect and we need to consider than
-when we analyze the data. Sometimes, we need to use some common sense and
-scientific insight as we examine the data - just as we would for field data to
-identify questionable values.
+Sometimes a dataset's metadata will tell us the range of expected values for a 
+variable or common sense dictates the expected value (as in the Julian day example 
+above). Values outside of this range are suspect and we need to consider them 
+when we analyze the data. Sometimes, we need to use some common sense and scientific 
+insight as we examine the data - just as we would for field data to identify 
+questionable values.
 
 We can explore the distribution of values contained within our data using the
-`hist` function which produces a histogram. Histograms are often useful in
+`geom_histogram()` function which produces a histogram. Histograms are often useful in
 identifying outliers and bad data values in our raster data.
 
-HIST of SMALLISH DATA SET WITH SOME OUTLANDISH VALUE
+```{r histogram, fig.cap="Histogram with a good bell curve and some values clustered at zero."}
+
+# create and view histogram
+ggplot() +
+  geom_histogram(data = RGB_HARV_df, aes(HARV_Ortho_wNA), bins = 40)
+
+```
+
+In `geom_histogram()`, 30 is the default number of bins for the data. We can define the number of bins by using the `bins` value. 
 
+The shape of this histogram looks reasonable and there are likely no bad data values in this raster.
```

Comment:
Am I right with this? There are a cluster of values at zero but I'm not sure if that necessarily means "bad data"

Reply:
I will merge this request so that I can fetch/merge and take a look at this line on my machine. More soon!